tests/checks/performance/performance_report_test.py

```diff
@@ -69,6 +70,26 @@ def test_classification(iris_split_dataset_and_model):
                 metric_col = class_col.loc[class_col['Metric'] == metric]
                 assert_that(metric_col['Value'] , close_to(1, 0.3))
 
+
+def test_classification_binary(iris_dataset_single_class_labeled):
+    # Arrange
+    train, test = train_test_split(iris_dataset_single_class_labeled.data, test_size=0.33, random_state=42)
+    train_ds = iris_dataset_single_class_labeled.copy(train)
+    test_ds = iris_dataset_single_class_labeled.copy(test)
+    clf = RandomForestClassifier(random_state=0)
+    clf.fit(train_ds.features_columns, train_ds.label_col)
+    check = PerformanceReport()
+
+    # Act X
+    result = check.run(train_ds, test_ds, clf).value
+    # Assert
+    for dataset in ['Test', 'Train']:
+        dataset_col = result.loc[result['Dataset'] == dataset]
+        for metric in ['Accuracy (Default)', 'Precision (Default)', 'Recall (Default)']:
```

Comment:
maybe use the default dict `keys()` for future changes

Reply:
fixed